data/plugins.json

```diff
@@ -157,23 +206,33 @@
     "safari": 10,
     "node": 6,
     "ios": 10,
+    "chrome-flag": null,
+    "edge-flag": 13,
+    "node-flag": 6,
     "opera": 37
   },
   "transform-exponentiation-operator": {
     "chrome": 52,
     "edge": 14,
     "firefox": 52,
+    "chrome-flag": 51,
+    "edge-flag": 14,
+    "node-flag": 6.5,
     "opera": 39
   },
   "transform-async-to-generator": {
     "chrome": 55,
     "firefox": 52,
+    "chrome-flag": 52,
+    "edge-flag": 13,
     "opera": 42
   },
   "syntax-trailing-function-commas": {
```

Comment:
(not related to this specific PR but) wait... a syntax plugin? Isn't this only used for enabling `babylon` features?

Or does it happen to work because of how it interacts with `babel-generate`? If that's the case, should not preserving the trailing comma be considered a `babel-generate` bug? 😄